### PR TITLE
Add /team page

### PR DESCRIPTION
**Pull request by jameshadfield**
This direction was discussed & agreed on in Nextstrain meetings in October / November 2022. While not implemented here, it would be nice to expand this to provide some brief content on (either or both) what parts people were (mainly) working on and where they are now.

Please add commits to this PR as you see fit so we can make it a team effort 😂 

The docs theme will also need updating - https://github.com/nextstrain/sphinx-theme/blame/e026ced8de3d2973f19e7c382f45e9e546b609ed/lib/nextstrain/sphinx/theme/footer.html

### CODE REVIEW

**Comment by emmahodcroft** [comment]
Thanks James, I think this looks great! I really like the 'team' page with the alumni section, too!

There's only one thing that I'd suggest - to separate out @rneher & @trvrb, perhaps a line above, to show their founder status. I think for Trevor it's not an issue, but I think Richard's role is sometimes overlooked (and I've had some questions asking about this in the past). Perhaps it could say something like

Nextstrain was founded by:
<Trevor> <Richard>
The Nextstrain core team is:
<Everyone>

If it's not thought that this would be too large a footer!

File: static-site/src/components/Footer/index.jsx [hunk 1]
```diff
@@ -65,29 +66,21 @@ class Footer extends React.Component {
...
+                  <div style={{margin: "0px 0px"}}/>
+                  <div style={{display: "flex", flexWrap: "wrap", justifyContent: "center"}}>
+                    {teamMembers.map((p, i) => 
+                      <TeamMember key={p.name} name={p.name} image={p.image} link={p.link} comma={i+1!==teamMembers.length}/>
+                    )}
+                  </div>
                   {"Please see our "}
-                  <Link to="/alumni">alumni page</Link>
+                  <Link to="/team">team page</Link>
                   {" for previous members of the team"}
```
Comment (tsibley) [inline static-site/src/components/Footer/index.jsx:81]:
```suggestion
                  {" for previous members of the team."}
```

File: static-site/src/pages/team.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  font-weight: 300;
+  padding: 20px 5%;
+  & img {
+    width: 60px;
+    margin-right: 20px !important;
+  }
+  & > span {
+    padding-bottom: 15px;
+  }
+`;
```
Comment (tsibley) [inline static-site/src/pages/team.jsx:56]: This grid layout feels a little lopsided to the left for me:

image

but maybe it works out better once we have a sentence or two?

File: static-site/src/pages/team.jsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import { CenteredFocusParagraph } from "../components/splash/styles";
+
+/**
+ * Todo - we would like a brief sentence or two describing what people work on
+ * (or worked on) here. We need to both write the sentence and then render this
+ * (only in the grid, not in the footer).
+ */
+
+export const teamMembers = [
+  {name: "Trevor Bedford", image: "trevor-bedford.jpg", link: "http://bedford.io/team/trevor-bedford/"},
```
Comment (tsibley) [inline static-site/src/pages/team.jsx]: Pushed a commit to make all the `http://` links `https://` instead.

File: static-site/src/pages/team.jsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  {name: "Ivan Aksamentov", image: "ivan-aksamentov.jpg", link: "https://neherlab.org/ivan-aksamentov.html"},
+  {name: "Jover Lee", image: "jover-lee.jpg", link: "http://bedford.io/team/jover-lee/"},
+  {name: "Jennifer Chang", image: "jennifer-chang.jpg", link: "https://bedford.io/team/jennifer-chang/"},
+  {name: "Victor Lin", image: "victor-lin.png", link: "https://bedford.io/team/victor-lin/"},
+  {name: "Cornelius Roemer", image: "cornelius-roemer.jpg", link: "https://neherlab.org/cornelius-roemer.html"},
+];
+
+const alumni = [
+  {name: "Cassia Wagner", image: "cassia-wagner.jpg", link: "https://bedford.io/team/cassia-wagner/"},
+  {name: "Miguel Parades", image: "miguel-parades.jpg", link: "https://bedford.io/team/miguel-parades/"},
```
Comment (tsibley) [inline static-site/src/pages/team.jsx:29]: It was at first a little weird to see active Blab members listed as alumni, but I think I understand why: they were actively contributing to SARS-CoV-2 analyses and narratives a few years ago, but no longer are, right?

**Review by tsibley — APPROVED** [review]
Looks alright to me. Thanks for doing this. A few non-blocking comments.

File: static-site/src/pages/team.jsx (hunk 2 above)
Comment (jameshadfield) [reply static-site/src/pages/team.jsx:56]: Yeah, I know what you mean. It _should_ be better with a sentence, let's see...

File: static-site/src/pages/team.jsx (hunk 4 above)
Comment (jameshadfield) [reply static-site/src/pages/team.jsx:29]: Yeah - I talked about this with Trevor & the rough thinking was core = those who are at the biweekly meetings. But happy either way - I agree it's a little strange!

**Comment by emmahodcroft** [comment]
Alternative suggestion for the 'bottom of page' version - just add `(Founder)` or `(Founder)` after Richard & Trevor's names

**Comment by jameshadfield** [comment]
An afternoon's work has drifted into a month-long PR!

I've corrected some typos, made some layout changes, and separated out the co-founders in the /team page. It now looks good enough to release, so I plan to merge today.

I think we all still want a little sentence for each person so I plan to make a new PR on top of this and then invite people to add their blurb (as I'm fearful that may take another month!).

**Pull request merged**